radicle-node/src/test/peer.rs

```diff
@@ -237,7 +238,20 @@ where
     }
 
     pub fn refs_announcement(&self, rid: Id) -> Message {
-        let refs = BoundedVec::new();
+        let mut refs = BoundedVec::new();
+        if let Ok(repo) = self.storage().repository(rid) {
+            if let Ok(false) = repo.is_empty() {
```

Comment:
Is this needed? I'd expect `remotes()` to just not return anything if it's empty.

Reply:
yeah it's needed, as some implementation of `remotes()` panics:  https://github.com/radicle-dev/heartwood/blob/de8e92d48cb4810ca7323f4468504d7968dd1a02/radicle/src/test/storage.rs#L116-L117